system/src/Grav/Common/Language/Language.php

```diff
@@ -472,12 +472,13 @@ public function getTranslation($lang, $key, $array_support = false)
     public function getBrowserLanguages($accept_langs = [])
     {
         if (empty($this->http_accept_language)) {
-            if (empty($accept_langs) && isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
-                $accept_langs = $_SERVER['HTTP_ACCEPT_LANGUAGE'];
-            } else {
+
+            if (empty($accept_langs) == false && isset($_SERVER['HTTP_ACCEPT_LANGUAGE']) == false) {
```

Comment:
Same as

```php
if (!empty($accept_langs) && !isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
```

This code changes the meaning of the code (resulting undefined variable $_SERVER['HTTP_ACCEPT_LANGUAGE']), use *or* instead:

```php
if (!empty($accept_langs) || !isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
```